examples/flax/text-classification/run_flax_glue.py

```diff
@@ -82,7 +87,7 @@ def parse_args():
         "--validation_file", type=str, default=None, help="A csv or a json file containing the validation data."
     )
     parser.add_argument(
```

Comment:
Could we also use the argument dataclasses here, to be consistent with other flax examples ?

Reply:
done